scimma_admin/scimma_admin/settings.py

```diff
@@ -118,24 +119,22 @@ def get_rds_db(db_instance_id):
     }
 
 
-# Password validation
-# https://docs.djangoproject.com/en/3.0/ref/settings/#auth-password-validators
+# Authentication
+# https://mozilla-django-oidc.readthedocs.io/en/stable/settings.html
 
-AUTH_PASSWORD_VALIDATORS = [
-    {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator',
-    },
-]
+AUTHENTICATION_BACKENDS = (
+    'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
+)
```

Comment:
This should probably be different when running locally. I don't know what the right Django auth backend is for local dev, though.

Reply:
with respect to local development, was there a registration process with cilogon.org? Is that the issue with local devel?

Reply:
Yeah, it's been a while since my last AuthN application, but my understanding is the issue is the inability to register the application with the auth provider, not necessarily this Auth Backend library. Maybe just a comment about local development instead?

Reply:
Right, OIDC auth works with redirects. In this case, Django redirects a user to `cilogon.org`, and pass in a "redirect_uri" query parameter which is set to "https://admin.dev.hop.scimma.org/auth/callback/". The oauth provider (cilogon, here) checks that redirect URI **exactly** to see that it matches the registered application; if it does (and the login is successful, of course) then the user gets sent back to that URI.

This won't work if you're doing localhost development, unless you register an OIDC application with a redirect URL of something like `http://localhost:8000/auth/callback/`. We could do that, I suppose.

Reply:
Yup, that's what I was thinking. And that's what we had done previously at a client, although we had four multiple environments--but for the dev auth provider at least, `localhost` was registered.